blocks/bricks/interfaces.py

```diff
@@ -141,31 +141,61 @@ class Initializable(RNGMixin, Brick):
 
     """
     has_biases = True
+    initialization_roles = set([])
 
     @lazy()
-    def __init__(self, weights_init=None, biases_init=None, use_bias=True,
-                 seed=None, **kwargs):
+    def __init__(self,
```

Comment:
Please, do not use one-keyword-argument-per-line indentation scheme. It is never used in Blocks, for the sake of maintaining a common style for the codebase, I would like to ask you not to use it as well.